bark/models/dynamic/integration.hpp

```diff
@@ -10,15 +10,21 @@
 #define BARK_MODELS_DYNAMIC_INTEGRATION_HPP_
 #include <Eigen/Core>
 
+#include "bark/geometry/angle.hpp"
 #include "bark/models/dynamic/dynamic_model.hpp"
 
 namespace bark {
 namespace models {
 namespace dynamic {
 
-inline State euler_int(const DynamicModel& model, const State& x,
-                       const Input& u, float dt) {
-  return x + dt * model.StateSpaceModel(x, u);
+inline State euler_int(const DynamicModel& model,
+                const State &x,
+                const Input &u,
+                float dt) {
+  State new_x = x + dt * model.StateSpaceModel(x, u);
+  new_x(StateDefinition::THETA_POSITION) =
```

Comment:
this should not be in the integration function. Why is this required now?

Reply:
it is to make sure theta is between [0 ... 2pi]. why not here?

Reply:
It somehow should not be in the integration function. A normalization function could be added. 
But also okay for me